about.html

```diff
@@ -12,62 +12,96 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
 
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
+  <div class="container">
+
+    <header>
+
+      <nav>
+          <img  src="img/lambda-black.png" alt="">
+         
+            <a href="index.html">Home</a>
+            <a href="about.html">About</a>
+            <a href="#">Products</a>
+            <a href="#">Blog</a>
+            <a href="contact.html">Contract</a>
+         </nav>
+             
+    </header>
+   
+        <div class="container about-page">
+            <img src="img/jumbo-about.png">
+     
+      <div class="text-container about">
+        <h2> About</h2>
 
+          <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+      </div>
+    </div>
 
+  <div class="top-content">
 
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+    <div class="text-container about-text">
+      <img src="img/about-plan.png" alt="strategy">
+      
+      <h2> Strategy</h2>
+        
+        <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
 
-    Learn More
+        <a href="#">Learn More</a>
```

Comment:
Unless you want something to link something on your site, it's not necessary to add an anchor tag, whose purpose is to provide a link. In this case, the Learn More buttons would have benefitted more from a <button> tag.